dev-tools/docker/ubuntu_devbox/Dockerfile

```diff
@@ -0,0 +1,193 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0 and the following additional limitation. Functionality enabled by the
+# files subject to the Elastic License 2.0 may only be used in production when
+# invoked by an Elasticsearch process with a license key installed that permits
+# use of machine learning features. You may not use this file except in
+# compliance with the Elastic License 2.0 and the foregoing additional
+# limitation.
+#
+
+FROM ubuntu:22.04 AS builder
+
+# This is basically automating the setup instructions in build-setup/linux.md
+
+MAINTAINER Valeriy Khakhutskyy <valeriy.khakhutskyy@elastic.co>
```

Comment:
I get this when I try to build? Idk if we want to switch to `label` or not, I'm on Ubuntu 22.04.5 LTS, Docker version 27.3.1

>  => WARN: MaintainerDeprecated: Maintainer instruction is deprecated in favor of using label (line 16)                                               0.0s